client/cz-component-dash.html

```diff
@@ -141,21 +149,23 @@
         return issueList.summary(updateSLO);
       },
 
-      updateIssueSource: function({tag, query}) {
-        registerSource(tag, query, issueList => {
-          this.set('issueList', issueList);
-        });
+      updateUsernames: function(userConfigs, issueList) {
+        if (!issueList) {
+          return;
+        }
+        var usernames = userConfigs.map(userConfig => issueList.getUsername(userConfig));
+        this.set('usernames', new Set(usernames));
       },
 
-      updateComponent: function(issueList) {
+      updateComponent: function(usernames, issueList) {
         var team = new IssueList();
         var others = new IssueList();
         var unowned = new IssueList();
         if (issueList) {
           for (var issue of issueList) {
             if (issue.owner == null) {
               unowned.push(issue);
-            } else if (this.users.indexOf(issue.owner) === -1) {
+            } else if (!usernames.has(issue.owner)) {
```

Comment:
FYI, you can use .includes on a list directly now, so you don't need to Set-ify usernames (I don't think there's any other reason to do that here as it isn't edited anywhere).